assets/training/model_management/src/azureml/model/mgmt/processors/transformers/convertors.py

```diff
@@ -175,7 +177,65 @@ def _save(
             else:
                 logger.info(f"Successully installed {pkgs}. pip logs =>\n{stdout}\n")
 
-        if segregate:
+        # Set experimental flag
+        if self._experimental:
+            logger.info("Experimental features enabled for MLflow conversion")
+            self._hf_conf["exp"] = True
+
+        try:
+            # create a conda environment for OSS transformers Flavor
+            python_version = platform.python_version()
+            pip_pkgs = self._get_curated_environment_pip_package_list()
+            conda_deps = CondaDependencies.create(conda_packages=None,
+                                                  python_version=python_version,
+                                                  pip_packages=pip_pkgs,
+                                                  pin_sdk_version=False)
+
+            curated_conda_env = conda_deps.as_dict()
+
+            # handle OSS trust_remote_code value
+            trust_remote_code_val = False
```

Comment:
can we make use of hf_pipeline_args?

Reply:
we have config, model, tokenizer and pipeline args right? that is meant to be used for passing any init params.

Reply:
looks like there is trust remote code handling here: https://github.com/Azure/azureml-assets/blob/2fee96555655df5e4ae2c18d1b7f1b839d6e8198/assets/training/model_management/src/run_model_preprocess.py#L144C1-L155C108


Could you please check if change here is also needed?

Reply:
The code mentioned above is setting the values, whereas the code added in the PR is checking that all the values if set are provided to the transformers pipeline